tools/image_processing/bia-ftplinks/biaftplink.xml

```diff
@@ -1,53 +1,55 @@
 <tool id="bia_download" name="FTP Link for Bioimage Archive" version="@VERSION@+galaxy0" profile="22.05">
-    <description>Download images from Bioimage Archive</description>
+    <description>Download images (TIFF) from Bioimage Archive</description>
     <macros>
         <import>macros.xml</import>
     </macros>
     <expand macro="requirements" />
     <command detect_errors="aggressive">
 <![CDATA[
-        wget -r 'ftp://ftp.ebi.ac.uk/biostudies/$mode/$path'/Files; 
-        #if '$ftp_output'
-            #set study = $path.split('/')[-1].rstrip('/')
-            curl https://www.ebi.ac.uk/biostudies/api/v1/studies/$study/info -s |jq -r .ftpLink >>ftpLink.txt
-        #end if
+    curl -s https://www.ebi.ac.uk/biostudies/api/v1/studies/$accession/info | jq -r .ftpLink >> ftpLink.txt &&
+    wget -q -r -l 0 -i ftpLink.txt &&
```

Comment:
How do we make sure that we only get tif files?

Reply:
We get the whole ftp directory of that accession.

Reply:
There is no particular directory structure. So I download everything and filter for tif later.

Reply:
Can you not filter this file first, instead of downloading everything and throwing that away later?